data-prepper-api/src/test/java/com/amazon/dataprepper/model/configuration/PipelinesDataFlowModelTest.java

```diff
@@ -36,13 +36,13 @@ void setup() {
     }
 
     @Test
-    void testSerializing_PipelinesDataFlowModel_empty_Plugins() throws JsonProcessingException {
+    void testSerializing_PipelinesDataFlowModel_empty_Plugins_with_nonEmpty_delay_and_workers() throws JsonProcessingException {
```

Comment:
How about still having a test with empty delay and workers, has that behavior changed?

Reply:
None of the behavior has changed. If they are null they won't be included in the serialized result. I could add a test for this, but it doesn't feel completely necessary. Before, we weren't testing them at all with non-null values

Reply:
I like the addition of testing with non-null values, including the null values test would just be more of a sanity check but it is not a blocker for this PR.